2020-01-Winter/3_stacks_queues/ValidParentheses

```diff
@@ -0,0 +1,52 @@
+  #include <iostream>
+  #include <stack>
+  
+  using namespace std;
+  
+  bool isValid(string s);
+  
+  int main() {
+    string test0 = "()"; // Output: true
+    string test1; = "()[]{}" //Output: true
+    string test2 = (]; // Output: false
+    string test3 = "{[]}; // Output: true
+    cout << "test0 = " << boolaplha << test0 << endl;
```

Comment:
What is `boolaplha`? Also, your tests never call the `isValid(string)` implementation. Compilation issue